basic/atm/ATM.py

```diff
@@ -0,0 +1,92 @@
+class ATM:
+    PATH_SAVING_FILE = "ATM_data_base.txt"
+
+    def __init__(self):
+        self._accounts_dict = {}
+        with open(ATM.PATH_SAVING_FILE, "r") as f:
+            account_lst = f.readlines()
+            for account in account_lst:
+                account = account.split()
+                self._accounts_dict[account[0]] = [account[1], int(account[2])]
+        self.main_atm()
+
+    def main_atm(self):
+        id_number = ""
+        password_flag = False
+        while not password_flag:
+            id_number = input("Welcome, enter id number\n")
+            if id_number in self._accounts_dict:
+
+                password = input("Enter password\n")
+                if self.check_password(id_number, password):
+                    password_flag = True
+                else:
+                    print("The password is not correct, try again.")
+            else:
+                print("This id is not in the system, try again.")
+
+        choice_flag = False
+        while not choice_flag:
+            choice = input("Hello,\n enter 1 - to check the balance\nEnter 2 - to deposit cash\n"
+                           "Enter 3 - for cash withdrawal\nEnter 4 - to change password\nEnter -1 - to exit ATM.\n")
+            if choice == '1':
+                print("Your balance is", self.check_the_balance(id_number))
+
+            elif choice == '2':
+                deposit = int(input("Enter the amount of money you want to deposit.\n"))
+                self.cash_deposit(id_number, deposit)
+                print("The deposit went successfully. Your balance is", self.check_the_balance(id_number))
+
+            elif choice == '3':
+                withdrawal = int(input("Enter the amount of money you want to withdrawal.\n"))
+                self.cash_withdrawal(id_number, withdrawal)
+                print("The withdrawal went successfully. Your balance is", self.check_the_balance(id_number))
+
+            elif choice == '4':
+                old_password_flag = False
+                while not old_password_flag:
```

Comment:
`old_password_flag` is not indicative - suggestion for a better name:
"requested_to_exit"